isthmus/src/test/java/io/substrait/isthmus/SubstraitToCalciteTest.java

```diff
@@ -0,0 +1,209 @@
+package io.substrait.isthmus;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import io.substrait.plan.ImmutableRoot;
+import io.substrait.plan.Plan;
+import io.substrait.plan.ProtoPlanConverter;
+import io.substrait.type.Type;
+import io.substrait.type.TypeCreator;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map.Entry;
+import org.apache.calcite.adapter.tpcds.TpcdsSchema;
+import org.apache.calcite.rel.RelRoot;
+import org.apache.calcite.rel.type.RelDataType;
+import org.apache.calcite.sql.type.SqlTypeName;
+import org.junit.jupiter.api.Test;
+
+public class SubstraitToCalciteTest extends PlanTestBase {
+  @Test
+  void testConvertRoot() throws Exception {
```

Comment:
might be nice to split this into separate test functions 😄

Reply:
sure. I can split it up